### PR TITLE
Fix calendar subsequent fetch timing with multiple calendar entries

### PR DESCRIPTION
**Pull request by sdetweil**
- Does the pull request solve a **related** issue?
#2160 
- If so, can you reference the issue?
- What does the pull request accomplish? Use a list if needed.
moves setInterval hander to inside addCalendar() and changes to setTimeout calling same routine
parameters are non-volatile

### CODE REVIEW

File: modules/default/calendar/calendar.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ Module.register("calendar", {
 		calendars: [
 			{
 				symbol: "calendar",
-				url: "https://www.calendarlabs.com/templates/ical/US-Holidays.ics"
+				url: "https://www.calendarlabs.com/templates/ical/US-Holidays.ics1"
```
Comment (rejas) [inline modules/default/calendar/calendar.js:44]: Typo I assume?

File: modules/default/calendar/node_helper.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ module.exports = NodeHelper.create({
 		} else {
 			Log.log("Use existing calendar fetcher for url: " + url);
 			fetcher = self.fetchers[identifier + url];
-			fetcher.broadcastEvents();
+			//fetcher.broadcastEvents();
```
Comment (rejas) [inline modules/default/calendar/node_helper.js:73]: Why is that  outcommented?

File: modules/default/calendar/calendar.js [hunk 3]
```diff
@@ -114,12 +116,12 @@ Module.register("calendar", {
 
 			this.addCalendar(calendar.url, calendar.auth, calendarConfig);
 
-			// Trigger ADD_CALENDAR every fetchInterval to make sure there is always a calendar
+			/*			// Trigger ADD_CALENDAR every fetchInterval to make sure there is always a calendar
```
Comment (rejas) [inline modules/default/calendar/calendar.js:119]: Why not remove the code if it isnt used here anymore?

File: modules/default/calendar/calendar.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ Module.register("calendar", {
 		timeFormat: "relative",
 		dateFormat: "MMM Do",
 		dateEndFormat: "LT",
-		fullDayEventDateFormat: "MMM Do",
+		fullDayEventDateFormat: "MMM Do HH:mm",
```
Comment (rejas) [inline modules/default/calendar/calendar.js:32]: Why is that necessary for a full day event?

File: CHANGELOG.md [hunk 5]
```diff
@@ -20,6 +20,7 @@ _This release is scheduled to be released on 2021-01-01._
 - JSON Parse translation files with comments crashing UI. (#2149)
 - Calendar parsing where RRULE bug returns wrong date, add Windows timezone name support. (#2145, #2151)
 - Wrong node-ical version installed (package.json) requested version. (#2153)
+- Fix calendar fetcher subsequent timing (#2160)
```
Comment (rejas) [inline CHANGELOG.md:23]: :+1:

Comment (sdetweil) [reply CHANGELOG.md:23]: sorry, change that should not have made it in, will fix

File: modules/default/calendar/calendar.js (hunk 1 above)
Comment (sdetweil) [reply modules/default/calendar/calendar.js:44]: yes, will fix, thanks

File: modules/default/calendar/calendar.js (hunk 3 above)
Comment (sdetweil) [reply modules/default/calendar/calendar.js:119]: will fix
